src/app/components/client/ProgressCard.tsx

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { CSSProperties, useState } from "react";
+import { QuestionMarkCircle } from "../server/Icons";
+import styles from "./ProgressCard.module.scss";
+import ExploringLaptopPlus from "./assets/exploring-laptop-check.svg";
+import ExploringLaptopMinus from "./assets/exploring-laptop-minus.svg";
+import SparklingCheck from "./assets/sparkling-check.svg";
+import Image from "next/image";
+import { Modal } from "./Modal";
+ 
+ export type Props = {
+   resolvedByYou: number;
+   autoRemoved: number;
+   totalNumExposures: number;
+ };
+ 
+ function PercentageComplete(props: Props) {
+  const totalRemoved = props.autoRemoved + props.resolvedByYou;
+
+   const percentageCompleteNum =
+   (totalRemoved > 0 && props.totalNumExposures > 0) ? ((props.autoRemoved + props.resolvedByYou) / props.totalNumExposures) * 100 : 0 // Prevents the division of 0
```

Comment:
Nit: This is getting pretty long. We can apply the formatting changes here and in other places as discussed offline in bulk together with https://github.com/mozilla/blurts-server/pull/3146.